spec/rubocop/cop/layout/closing_heredoc_indentation_spec.rb

```diff
@@ -140,5 +150,19 @@ def foo
         end
       RUBY
     end
+
+    it 'does not correct a << heredoc' do
+      source = <<-RUBY.strip_indent
+        class Test
+          def foo
+            <<SQL
+              bar
+        SQL
+          end
+        end
+      RUBY
+
+      expect(autocorrect_source(source)).to eq source
+    end
```

Comment:
Probably this test case is unnecessary. Previously, I was pointed out as follows 😃 

> `expect_no_offenses` is enough to check this cop does not auto-correct.

https://github.com/rubocop-hq/rubocop/pull/4764#discussion_r140652557

Reply:
Indeed. If something doesn't generate an offense it's certainly not going to do auto-correct.

Reply:
Ok will make the change.